MultiSelect1/clients/MultiSelect1/controllers/page/Start/onStart.js

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Generally weird indentation here, looks like everything is indented to start which probably isn't necessary.

Code inside function defs should be indented, function defs should all be indented the same, etc...